Source/WebCore/bindings/js/JSDOMWindowBase.cpp

```diff
@@ -304,17 +305,17 @@ void JSDOMWindowBase::reportViolationForUnsafeEval(JSGlobalObject* object, JSStr
 
 String JSDOMWindowBase::codeForEval(JSGlobalObject* globalObject, JSValue value)
 {
-    VM& vm = globalObject->vm();
-
-    if (auto* script = JSTrustedScript::toWrapped(vm, value))
-        return script->toString();
+    return JSDOMGlobalObject::codeForEval(globalObject, value);
```

Comment:
Why do we need to wrap this while JSDOMWindowBase's base class is JSDOMGlobalObject?

Reply:
I was following the pattern of the surrounding code but you're right I don't need to do any of this stuff it just works with the definitions in JSDOMGlobalObject. Will clean that up.